rust/perspective-viewer/exprtk.md

```diff
@@ -37,6 +37,22 @@ Bucket x by y
 bucket(${1:x}, ${2:y})
 ```
 
+When `x` is numeric, `y` should also be a number.
+
+When `y` is a `date` or a `datetime`, `y` can be one of:
+
+-   `s` -- seconds
+-   `m` -- minutes
+-   `h` -- hours
+-   `D` -- days
+-   `W` -- weeks
+-   `M` -- months
+-   `Y` -- years
+
+In all of these except for days and weeks, a non-negative number can be placed before the unit character.
+For example, `3M` would bucket by 3 months. These must divisible numbers; you can not bucket by
+`7M` or `22s`, but by `30m`, `12h`, etc.
+
```

Comment:
This file is auto-generated from the [rust source](https://github.com/finos/perspective/blob/emsdk-update/rust/perspective-viewer/src/rust/exprtk/language.rs#L42). Currently, this same text is used to generate the auto-complete hints and it does not support markdown, so this text is not currently suitable there either.

We'll need to add something like a short/long text description field to the exprtk language spec to support this.

<img width="479" alt="Screenshot 2023-04-29 at 5 26 39 PM" src="https://user-images.githubusercontent.com/60666/235324952-80b0772d-0040-4679-8cee-c19d0ab2479a.png">